aws/table_aws_dax_parameter.go

```diff
@@ -0,0 +1,150 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+
+	"github.com/aws/aws-sdk-go-v2/service/dax"
+	"github.com/aws/aws-sdk-go-v2/service/dax/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsDaxParameter(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_dax_parameter",
+		Description: "AWS DAX Parameter",
+		List: &plugin.ListConfig{
+			ParentHydrate: listDaxParameterGroups,
+			Hydrate:       listDaxParameters,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ParameterGroupNotFoundFault"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "parameter_group_name",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "parameter_name",
+				Description: "The name of the parameter.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "parameter_group_name",
+				Description: "The name of the parameter group.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "parameter_value",
+				Description: "The value for the parameter.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "description",
+				Description: "Description of the parameter.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "allowed_values",
+				Description: "A range of values within which the parameter can be set.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "change_type",
+				Description: "The conditions under which changes to this parameter can be applied.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "data_type",
+				Description: "The data type of the parameter.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "is_modifiable",
+				Description: "Whether the customer is allowed to modify the parameter.",
+				Type:        proto.ColumnType_STRING,
```

Comment:
Shouldn't this be of the boolean type? From the description, it looks as if it would have a true/false value.

Reply:
The data type will be string types, and the value that we are getting from the response is in string format, not in boolean format i.e. TRUE/FALSE